examples/cart/index.js

```diff
@@ -158,13 +160,37 @@ $(function() {
   ============================================================ */
   function openCart() {
     $('.cart').addClass('js-active');
+    $('.overlay').addClass('js-active');
   }
 
   /* Close Cart
   ============================================================ */
   function closeCart() {
-    $('.cart .btn--close').click(function () {
-      $('.cart').removeClass('js-active');
+    $('.cart').removeClass('js-active');
+    $('.overlay').removeClass('js-active');
+  }
+
+  /* Close Cart Handler
+  ============================================================ */
+  function closeCartHandler() {
+    $('.cart .btn--close').click(closeCart);
```

Comment:
I think `.on('click', ...` is preferred here, though it looks like it's no longer explicitly stated in the JS style guide. @lemonmade, was that removed intentionally or just lost in the migration?